### PR TITLE
new README using the devkit template

### PR DESCRIPTION
**Pull request by nweldev**
New README based on the [devkit README template](https://github.com/blindnet-io/openness-framework/blob/main/OpenSource/template/devkit-README.md).

see https://github.com/blindnet-io/devrel-management/issues/9 & https://github.com/blindnet-io/openness-framework/pull/33

:warning: @blindnet-io/engineering  should still add the installation steps

API Ref can be addressed after this PR, but is a top priority.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,92 @@
...
+  <a href="https://github.com/blindnet-io/sdk-php-server/issues">Submit an Issue</a>
+  &nbsp;•&nbsp;
+  <a href="https://join.slack.com/t/blindnet/shared_invite/zt-1arqlhqt3-A8dPYXLbrnqz1ZKsz6ItOg">Online Chat</a>
+  <br>
+  <br>
+</p>
+
+## About
+
+This is the Java [server-side SDK](https://blindnet.dev/docs/extra/glossary#server-side-sdk) component of [blindnet devkit][devkit] allowing you to:
```
Comment (m4rk055) [inline README.md:29]:
```suggestion
This is the Java [server-side SDK](https://blindnet.dev/docs/glossary#server-side-sdk) component of [blindnet devkit][devkit] allowing you to:
```

File: README.md [hunk 2]
```diff
@@ -1 +1,92 @@
...
+
+- Generate [authentication tokens](https://blindnet.dev/docs/extra/glossary#authentication-token) to allow users to access blindnet.
```
Comment (m4rk055) [inline README.md:31]:
```suggestion
- Generate [authentication tokens](https://blindnet.dev/docs/glossary#authentication-token) to allow users to access blindnet.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,92 @@
...
+Use [{package manager or plateform}][install-tool] to install {project's name}:
+
+```bash
+{install-command}
+```
+
+<!-- FIXME: add API Reference
+## Usage
+📑 The API reference of blindnet devkit Node.js Server SDK is available on [blindnet.dev](https://docs.blindnet.io/docs/api_reference/server/node.js/latest).
+-->
```
Comment (noelmace) [inline README.md:50]: TODO @blindnet-io/engineering

**Review by nweldev — COMMENTED** [review]
TODO List for @blindnet-io/engineering before merging this in order to close https://github.com/blindnet-io/devrel-management/issues/9

File: README.md (hunk 3 above)
Comment (veselinovicst) [reply README.md:50]: There is no API reference for Java SDK.

**Comment by veselinovicst** [comment]
@noelmace resolved todo's

**Comment by noelmace** [comment]
@veselinovicst where? I don't see any new commit in this branch.

File: README.md (hunk 3 above)
Comment (noelmace) [reply README.md:50]: 1. You still need to complete the "install steps" section
2. There should be an API reference. Why didn't you add one? (we'll address that in another issue and PR later)
3. Only the person who made a review comment should resolve it, to make sure it has really been addressed.

**Pull request merged**